events/importer/kulke.py

```diff
@@ -179,6 +180,40 @@ def get_event_name(event):
             return names[0]
 
 
+def parse_age_range(secondary_headline):
+    if not isinstance(secondary_headline, str):
+        return (None, None)
+
+    pattern = r'^\D*(\d{1,2}).(\d{1,2}).[v.|år].+$'
+    match = re.match(pattern, secondary_headline)
+
+    if match:
+        beginning_age = int(match.groups()[0])
+        end_age = int(match.groups()[1])
+        return (beginning_age, end_age)
+    else:
+        return (None, None)
+
+
+def parse_course_time(secondary_headline):
+    if not isinstance(secondary_headline, str):
+        return (None, None)
+
+    pattern = r'^.*klo?\s(\d{1,2})\.?(\d{1,2})?.(\d{1,2})\.?(\d{1,2})?.*$'
```

Comment:
Here, I would suggest
`^.*klo?\s(\d{1,2})([.:](\d{1,2}))?.(\d{1,2})([.:](\d{1,2}))?.*$`
(and corresponding changes in group indices).